src/zodbupdate/tests.py

```diff
@@ -216,10 +218,10 @@ def tearDown(self):
 
     def _tearDownStorage(self):
         """
-        Called during *tearDown* to clean up storage-specific artifacts.
+        Called during *tearDown* to clean up storage-specific artefacts.
```

Comment:
Fun fact: In US English "artifact" is correct. Artefact is correct for British English.